resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/k8s/submit/submitsteps/DriverServiceBootstrapStepSuite.scala

```diff
@@ -108,7 +91,7 @@ private[spark] class DriverAddressConfigurationStepSuite
   }
 
   test("Long prefixes should switch to using a generated name.") {
-    val configurationStep = new DriverAddressConfigurationStep(
+    val configurationStep = new DriverServiceBootstrapStep(
         LONG_RESOURCE_NAME_PREFIX,
```

Comment:
this should be indented 2 spaces not 4 (same for the ones above).

Reply:
Done.